semantics.json

```diff
@@ -342,6 +342,14 @@
         "default": true
       },
       {
+        "name": "randomizeOnRetry",
+        "type": "boolean",
```

Comment:
Oh, how important small word fragments such as "re-" can be. I guess I am a little overworked. I just thought about this again, and my answer from two days ago made no sense at all. Yes, why would someone want to **re-**run the same order?!
I am really sorry. I'll just leave it with the single boolean option for randomization on/off which would mean either never randomize at all or randomize at start and on retry.